src/pocketmine/Player.php

```diff
@@ -1086,7 +1086,7 @@ public function dataPacket(DataPacket $packet, $needACK = false){
 		$timings = Timings::getSendDataPacketTimings($packet);
 		$timings->startTiming();
 
-		$this->server->getPluginManager()->callEvent($ev = new DataPacketSendEvent($this, $packet));
+		($ev = new DataPacketSendEvent($this, $packet))->call();
 		if($ev->isCancelled()){
```

Comment:
What about also returning isCancelled() in the return value, or a constant `false` if the event is not a Cancellable?